cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -386,24 +395,63 @@ def _translate_case(self, submission_date=None):
                 }
             ]
         }
-        if hasattr(self.case_data, "facts"):
-            request_data['applicant'].update(translate_applicant(self.case_data.facts))
-        if hasattr(self.case_data, "facts"):
-            request_data.update(translate_dependants(submission_date, self.case_data.facts))
-        if hasattr(self.case_data, "you") and hasattr(self.case_data.you, "savings"):
-            request_data.update(translate_savings(self.case_data.you.savings))
-        if hasattr(self.case_data, "property_data"):
-            request_data.update(translate_property(self.case_data.property_data))
-        if hasattr(self.case_data, "you") and hasattr(self.case_data.you, "income") and hasattr(self.case_data.you, "deductions"):
-            request_data.update(translate_employment(self.case_data.you.income, self.case_data.you.deductions))
-        if hasattr(self.case_data, "you") and hasattr(self.case_data.you, "income"):
-            request_data.update(translate_income(self.case_data.you.income))
-        if hasattr(self.case_data, "facts"):
-            request_data['applicant'].update(translate_age(submission_date, self.case_data.facts))
+        if hasattr(case_data, "facts"):
+            request_data['applicant'].update(EligibilityChecker._applicant_data(submission_date, case_data.facts))
+        if hasattr(case_data, "facts"):
+            request_data.update(translate_dependants(submission_date, case_data.facts))
+
+        request_data.update(EligibilityChecker._capital_data(case_data))
+        if not EligibilityChecker._complete_cfe_capital_data(request_data):
+            request_data['assessment']['section_capital'] = 'incomplete'
+
+        if hasattr(case_data, "you"):
+            request_data.update(EligibilityChecker._income_data(case_data.you))
+        if not EligibilityChecker._complete_cfe_income_data(request_data):
+            request_data['assessment']['section_gross_income'] = 'incomplete'
+
+        return request_data
+
+    @staticmethod
+    def _income_data(person):
+        request_data = {}
+        if hasattr(person, "income") and hasattr(person, "deductions"):
+            request_data.update(translate_employment(person.income, person.deductions))
+        if hasattr(person, "income"):
+            request_data.update(translate_income(person.income))
+        return request_data
+
+    @staticmethod
+    def _applicant_data(submission_date, facts):
+        request_data = {}
+        request_data.update(translate_age(submission_date, facts))
+        request_data.update(translate_applicant(facts))
+
+        return request_data
+
+    @staticmethod
+    def _capital_data(case_data):
```

Comment:
Let's have "translator" in the name, to indicate that it converts the capital data in case_data to a CFE-request dict.